src/main/java/org/jenkinsci/maven/plugins/hpi/HpiMojo.java

```diff
@@ -133,6 +134,14 @@ private void performPackaging()
             archiver.setArchiver(jarArchiver);
             archiver.setOutputFile(jarFile);
             jarArchiver.addConfiguredManifest(manifest);
+            File indexJelly = new File(getClassesDirectory(), "index.jelly");
+            if (!indexJelly.isFile()) {
+                throw new MojoFailureException("Missing " + indexJelly + ". Delete any <description> from pom.xml and create src/main/resources/index.jelly:\n" +
```

Comment:
Why do we want to delete the description from the pom? It makes sense to have a description in the pom as well.

Reply:
You do not _have_ to delete `<description>`; it would just be redundant and so there is a chance someone would try to make some text edits expecting them to appear in the product and be confused when they do not.